src/server/utils/datacube-guard/datacube-guard.mocha.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2019 Wirtualna Polska Media S.A.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Request } from "express";
+import { checkAccess } from "./datacube-guard";
+
+const assert = require("assert");
+
+describe("Guard test", () => {
+  it("Guard off -> access to all dataCubes", () => {
```

Comment:
I would add example with guard off, header for cube A and accessing cube B. It should pass. Contrived but will cover all scenarios.